test-runners/karma.js

```diff
@@ -41,28 +41,9 @@ function runCodeCoverage() {
         argv['code-coverage-threshold-statements'];
     }
 
-    if (argv['browserstack-username']) {
-      process.env.BROWSER_STACK_USERNAME = argv['browserstack-username'];
-    }
-
-    if (argv['browserstack-access-key']) {
-      process.env.BROWSER_STACK_ACCESS_KEY = argv['browserstack-access-key'];
-    }
-
-    if (argv['browserstack-build-id']) {
-      process.env.BROWSER_STACK_BUILD_ID = argv['browserstack-build-id'];
-    }
-
-    if (argv['browserstack-project']) {
-      process.env.BROWSER_STACK_PROJECT = argv['browserstack-project'];
-    }
-
     const result = crossSpawn.sync(
-      'npx',
+      'ng',
```

Comment:
I'm inclined to leave `npx` to avoid introducing any path issues. Is there a reason to remove it?

Reply:
Yes; for a few reasons. If we use `npx` it's wise to set the version so that the locally installed version of Angular CLI is not used (we don't want to rely on the consumer's package.json since it could be invalid). However, I'd prefer not to set the version in two places: here and on the pipeline. Additionally, we know the pipeline installs Angular CLI globally, so it's safe to use `ng` directly, here.